io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/GeoJSONInstanceWriter.java

```diff
@@ -45,20 +57,16 @@ public boolean isCancelable() {
 	protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
```

Comment:
We can remove this method, no additional value here compared to the super class.

Reply:
True, now writing the `TypeName` in the progress message using `getDefaultTypeName`.